tools/local_fix_format.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Runs the Enovy format fixers on the current changes. By default, this runs on
+# changes that are not yet committed. You can also specify:
+#
+#   -all: runs on the entire repository (very slow)
+#   -main: runs on the changes amde from the main branch
+#   file1 file2 file3...: runs on the specified files.
+#
+# To effectively use this script, it's best to run the format fixer before each
+# call to 'git commit'.  If you forget to do that then you can run with "-main"
+# and it will run on all changes you've made since branching from main, which
+# will still be relatively fast.
+#
+# To run this script, you must provide several environment variables:
+#
+# CLANG_FORMAT     -- points to the clang formatter binary, perhaps found in
+#                     $CLANG_TOOLCHAIN/bin/clang-format
+# BUILDIFIER_BIN   -- buildifier location, perhaps found in $HOME/go/bin/buildifier
+# BUILDOZER_BIN    -- buildozer location, perhaps found in $HOME/go/bin/buildozer
+
+# If DISPLAY is set, then tkdiff pops up for some BUILD changes.
+unset DISPLAY
+
+# Trigger an error if the required env vars are not set.
```

Comment:
the script `./tools/code_format/check_format.py` did that also, it also prints a message for reminding set the required env. so we can ignore this error here.

Reply:
done -- though somewhat reluctantly because I think the message previously printed was better.